pkg/features/kube_features.go

```diff
@@ -1273,6 +1273,28 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	genericfeatures.ZeroLimitedNominalConcurrencyShares: {Default: false, PreRelease: featuregate.Beta},
 
+	genericfeatures.APIServerIdentity: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
We may add `genericfeatures` better in alphabetic order from line 1244. 

https://github.com/kubernetes/kubernetes/blob/7314282b4c9efaba3b40cd557cb9384e6a9d160a/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L269-L330 
is not in alphabetic order  anyway.

Reply:
I've adjusted the feature gates order.